### PR TITLE
use base new name spaces

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
This version doesn't allow a multi-robot use. We can define < group ns="base" > but I don’t know if it will cause remap errors.

**Comment by floweisshardt** [comment]
LGTM

File: cob_navigation_global/launch/2dnav_linear.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
   <node pkg="cob_linear_nav" type="cob_linear_nav" respawn="false" name="move_base_linear" output="screen">
     <!-- remap inputs -->
-    <remap from="odom" to="/base_controller/odometry"/>
+    <remap from="odom" to="/base/odometry_controller/odometry"/>
     <!-- remap outputs -->
-    <remap from="cmd_vel" to="/base_controller/command_safe"/>
+    <remap from="cmd_vel" to="/base/twist_controller/command_safe"/>
```
Comment (fmessmer) [inline cob_navigation_global/launch/2dnav_linear.xml]: remove `/` prefix
(maybe also comments - for consistency with the other launch files)

**Comment by fmessmer** [comment]
Why are some `cmd_vel` remap'ed to `command`, some to `command_direct` and some to `command_safe`? Shouldn't they all be the same?

**Comment by fmessmer** [comment]
regarding multi-robot support, see [this conclusion](https://github.com/ipa320/care-o-bot/issues/11#issuecomment-147974410)

**Comment by floweisshardt** [comment]
'''command_safe''' for linear nav is correct as this navigation mode has no obstacle avoidance buid-in. 

BUT: for current cob4 footprint cob_base_velocity_filter cannot be use due to not rect-angular footprint.

**Comment by fmessmer** [comment]
Needs to be discussed in the group...see https://github.com/ipa320/cob_robots/pull/337

**Comment by fmessmer** [comment]
We should revisit this PR after https://github.com/ipa320/cob_robots/pull/346 has been merged and namespaces are clear...probably with input from @ipa-mig

**Comment by benmaidel** [comment]
I think we should merge this before doing anything in ipa320/cob_robots#346.
@ipa-nhg  have you tested all navigation launch files? For me it's also not clear why we have this different command topics...

**Comment by fmessmer** [comment]
@ipa-fmw's comment makes sense...
but then I'd prefer an non-functioning linear_nav for cob4 over an unsafe linear_nav for **ALL** robots

Thus, I think cmd_vel should be remapped to command_safe for linear_nav and remapped to command 
for all other launch files! Or what is the reason for the remap to command_direct for the eband_nav

**Comment by mgruhler** [comment]
To shed some light on this:
- `command_safe` goes into the `cob_collision_velocity_filter` which prevents driving the robot into an obstacle with teleop or linear nav. However, this currently only works with rectangular footprints
- `command` goes into the `cob_base_velocity_smoother` which is required for cob3 and r@w (thus, for cob4 probably as well) and provides smoother velocity commands. This was mainly due to the very hard commands from the standard ROS navigation (tr or dwa). The PR2 seemed to have a less stiff mechanics, why they didn't have problems with that. Could probably be reinvestigated...
- `command_direct` is actually the input into the base controller. the eband provides smoother outputs, this is why it goes directly into this.

**Comment by floweisshardt** [comment]
remap is done in https://github.com/ipa-nhg/cob_navigation/pull/1. @ipa-nhg: please merge and afterwards we can merge this PR

**Pull request merged**